fastestimator/trace/metric/bleu_score.py

```diff
@@ -0,0 +1,174 @@
+# Copyright 2022 The FastEstimator Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+from typing import Counter, Iterable, Union
```

Comment:
normally Counter would be imported from collections. are there any performance implications from taking it from typing?

Reply:
No are no performance implications, we can directly using the Counter from collections.

Reply:
If we're not referencing it in any type hints let's use the collections one